src/osemgrep/cli_scan/Scan_CLI.ml

```diff
@@ -613,11 +621,11 @@ let cmdline_term : conf Term.t =
    * of the corresponding '$ o_xx $' further below! *)
   let combine autofix baseline_commit config debug dryrun dump_ast dump_config
       emacs error exclude exclude_rule_ids force_color include_ json lang
-      max_memory_mb max_target_bytes metrics num_jobs optimizations pattern
-      profile project_root quiet replacement respect_git_ignore rewrite_rule_ids
-      scan_unknown_extensions severity show_supported_languages strict
-      target_roots test test_ignore_todo time_flag timeout timeout_threshold
-      validate verbose version version_check vim =
+      max_memory_mb max_target_bytes metrics num_jobs nosem optimizations
```

Comment:
o is before u in the alphabet usually :)

Reply:
@aryx could semgrep be used to enforce the alphabetic ordering of lists?